tests/console/yast2_kdump_use_fadump.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Select Firmware-Assisted Dump and accept kdump options.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "y2_module_consoletest";
+use strict;
+use warnings;
+
+use cfg_files_utils qw(validate_cfg_file);
+use scheduler qw(get_test_suite_data);
+use testapi qw(save_screenshot select_console);
+use utils qw(systemctl zypper_call);
+use YaST::Module;
+
+sub run {
+    my $firmware_assisted_dump = $testapi::distri->get_kdump_firmware();
```

Comment:
(optional) we could use fadump everywhere, methods, classes etc, and only include the full name in summaries, but it is fine as it is now.